packages/webapp/src/__tests__/WithdrawModal.test.ts

```
@@ -0,0 +1,259 @@
+	});
+
+	it('calls onSubmit with the amount when withdraw button is clicked', async () => {
+		render(WithdrawModal, defaultProps);
+
+		await waitFor(() => {
+			expect(screen.getByText('Balance of connected wallet')).toBeInTheDocument();
+		});
+
+		const inputAmount = '1';
+		const expectedAmountBigInt = BigInt(
+			parseFloat(inputAmount) * 10 ** Number(mockVault.token.decimals)
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)